src/services/hasura/hasura.service.ts

```
@@ -307,128 +350,160 @@ export class HasuraService {
-        const insertResult = await this.queryDb(query, item);
-        if (insertResult.errors) {
-          insertApiRes.push({ error: insertResult.errors, item_id: item.item_id });
-        } else {
-          insertApiRes.push(insertResult);
-        }
-      } catch (error) {
-     
-        insertApiRes.push({ error: error.message, item_id: item.item_id });
-      }
-    }
+		let insertApiRes = [];
```

ESLint

[error] 'insertApiRes' is never reassigned. Use 'const' instead.

(prefer-const)